docs/sections/user_guide/yaml/components/ww3.rst

```diff
@@ -0,0 +1,32 @@
+.. _ww3_yaml:
+
+ww3
+===
+
+Structured YAML to configure WaveWatchIII as part of a compiled coupled executable is validated by JSON Schema and requires the ``ww3:`` block, described below.
+
+Here is a prototype UW YAML ``ww3:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/ww3.yaml
+
+UW YAML for the ``ww3:`` Block
+------------------------------
+
+namelist:
+^^^^^^^^^
+
+.. important:: Fortran namelists for WaveWatchIII are used here as templates to be realized. Input namelists will not have their configurations validated, only the associated YAML. Other files are not currently supported.:
+
+  **template_file:**
+
+  The path to the base ``ww3_shel.nml`` file that comes from the WW3 build.
```

Comment:
```suggestion
  The path to the input template file containing Jinja2 expressions, based on the ``ww3_shel.nml`` file from the WaveWatchIII build.
```
True?

Reply:
Currently this is pointing directly to the namelist `ww3_shel.nml`.

Reply:
How is it currently pointing to anything? The user has to set `template_file:` in their UW YAML to specify the input template file they want us to use, right? We'd just be saying here that they should specify a file that is _based on_ `ww3_shel.nml` from the WaveWatch III build, but that they have (or their app maintainer has) updated to contain Jinja2 expressions.

Reply:
It is currently hard-coded within `ww3.py` as such, but perhaps we should explicitly call it `ww3_shel.nml.IN` to clearly differentiate it?

Reply:
Oh, right -- the _output_ filename is hardcoded, but the input filename is arbitrary. Given that we use the filename `ww3_shel.nml.IN` in our `shared/ww3.yaml`, how about e.g.
```
The path to the input template file containing Jinja2 expressions (perhaps named ``ww3_shel.nml.IN``), based on the ``ww3_shel.nml`` file from the WaveWatchIII build.
```
? It's not IMO really any of our business what they name their input template file, but if this makes the meaning clearer, I'm all for it.